src/components/Filters/FilterChecklist.vue

```diff
@@ -1,147 +1,101 @@
 <template>
-  <div
-    :class="{ ['filters']: true, ['single']: isSingleFilter, 'mb-8': true }"
-    @click="hideLicenseExplanationVisibility()"
-    @keyup.enter="hideLicenseExplanationVisibility()"
-  >
+  <fieldset class="mb-8" @click.stop="hideLicenseExplanationVisibility">
```

Comment:
Technically if a non-tabbable element has a click handler you're supposed to add `enter` and `space` as well as a tabindex of 0 to make it accessible, but given this is just a stop-gap solution until we replace it with the VPopover component I think this is okay 🤔

Reply:
Oh, I didn't realize that it was a non-tabbable element when I removed the `keyup.enter` event handling. And the `tabindex` wasn't there at all. Can't wait for VPopover replacement :)